pkg/subctl/cmd/validate_deployment.go

```diff
@@ -140,3 +145,26 @@ func CheckDaemonset(k8sClient kubernetes.Interface, namespace, daemonSetName str
 
 	return true
 }
+
+func checkPodsStatus(k8sClient kubernetes.Interface, operatorNamespace string) bool {
+	pods, _ := k8sClient.CoreV1().Pods(operatorNamespace).List(metav1.ListOptions{})
+	for _, pod := range pods.Items {
+		if pod.Status.Phase != v1.PodRunning {
+			message := fmt.Sprintf("Pod %q is not running. (current state is %v)", pod.Name, pod.Status.Phase)
+			status.QueueFailureMessage(message)
+			status.End(cli.Failure)
+			return false
+		}
+
+		for _, c := range pod.Status.ContainerStatuses {
+			if c.RestartCount >= 5 {
+				message := fmt.Sprintf("Pod %q is constantly restarting (restart count: %d)", pod.Name, c.RestartCount)
```

Comment:
This may not mean it's constantly restarting - it may have restarted 5 times over a period of hours/days/weeks for various reasons.  Perhaps we should report it as a warning if `RestartCount` > 0: "_Pod %q has restarted %d times_".